cluster/gce/util.sh

```diff
@@ -2082,6 +2093,48 @@ function create-etcd-certs {
   popd
 }
 
+# Generates SSL certificates for etcd-client and kube-apiserver communication. Uses cfssl program.
+#
+# Assumed vars:
+#   KUBE_TEMP: temporary directory
+#
+# Args:
+#  $1: host server name
+#  $2: host client name
+#  $3: CA certificate
+#  $4: CA key
+#
+# If CA cert/key is empty, the function will also generate certs for CA.
+#
+# Vars set:
+#   ETCD_APISERVER_CA_KEY_BASE64
+#   ETCD_APISERVER_CA_CERT_BASE64
+#   ETCD_APISERVER_SERVER_KEY_BASE64
+#   ETCD_APISERVER_SERVER_CERT_BASE64
+#   ETCD_APISERVER_CLIENT_KEY_BASE64
+#   ETCD_APISERVER_CLIENT_CERT_BASE64
+#
+function create-etcd-apiserver-certs {
+  local hostServer=${1}
+  local hostClient=${2}
+  local etcd_apiserver_ca_cert=${3:-}
```

Comment:
I don't think you want to allow those variables to be unset, as you're depending on them a line below it.

Reply:
@gmarek I think it would be fine, as in [generate-etcd-cert](https://github.com/kubernetes/kubernetes/blob/7f23a743e8c23ac6489340bbb34fa6f1d392db9d/cluster/common.sh#L347), "If GEN_ETCD_CA_CERT or GEN_ETCD_CA_KEY is not specified, it will generates certs for CA." The existing [create-etcd-certs](https://github.com/kubernetes/kubernetes/blob/e6413d20427cb78c20e3e3d963682a9413244db3/cluster/gce/util.sh#L2069) is set this way too. Is it OK?